Controllers/Mobile/v1/AuthController.cs

```diff
@@ -24,7 +24,7 @@ public async Task<IActionResult> ResendOtpAsync([FromBody] PhoneNumberReq phoneN
             if (!ModelState.IsValid)
                 return CustomBadRequest();
 
-            var storedOTP = await _authService.FindOtpAsync(phoneNumberReq.PhoneNumber);
+            var storedOTP = await _authService.GetOtpByPhoneNumber(phoneNumberReq.PhoneNumber);
```

Comment:
Please, prefix every method that queries the database for a record with `Find`. As this is the convention used by the rest of the contributors.